### DISCUSSION TITLE
Where are "slim" debian images?

### DISCUSSION THREAD

**Discussion by simon-liebehenschel** [docker-library/mongo#647] [General]
I am very sorry if this is a stupid question, but I wonder why I can find only `FROM debian` but no `slim` images (e.g. [bookworm-slim](https://hub.docker.com/layers/library/debian/bookworm-slim/images/sha256-abbf1e0df2d9631707a41780bd9d332523d10cbb14560122536210298b77f09d?context=explore)).

`slim` are listed in https://github.com/docker-library/mongo/blob/0bd2003f580af67939c634dc9a3775e3a4af7808/versions.json#L16 but it seems that you do not build them.

Are there any not so fat MongoDB images available? Maybe unofficial?

**Comment by tianon** [comment]
See https://github.com/docker-library/mongo/blob/ff232d138b9896900d6223c251e7413779d066af/versions.sh#L198-L201

The logic in our script scrapes all the supported versions (and converts those to relevant base images), but the final result is that we only publish/maintain the "most supported" one (where "most supported" is defined by the logic I've linked to there, generally most supported architectures upstream, which is why we end up with Ubuntu).
